### ISSUE TITLE
S3 - Http download url is not well-formed for default region (us-east-1 N. Virginia)

### ISSUE THREAD

**Issue by rodricifuentes1** [akka/alpakka#171]
When creating an S3 bucket in amazon located in N. Virginia, the url does not contain the bucket zone. 

Assuming that this behavior is only occurring for that region (because it is the default one) the correct host should be `s3.amazonws.com`

I added a condition in `HttpRequests` object just to test if my assumption was ok but I'm not sure if this behaviour could happend in any other regions 
```scala
def requestHost(s3Location: S3Location, region: String)(implicit conf: S3Settings): Uri.Host =
      conf.proxy match {
        # quick fix
        case None if region == "us-east-1" => Uri.Host(s"${s3Location.bucket}.s3.amazonaws.com")
        case None => Uri.Host(s"${s3Location.bucket}.s3-${region}.amazonaws.com")
        case Some(proxy) => Uri.Host(proxy.host)
      }
...
}
```

**Comment by pritesh-patel-hs** [comment]
Having the same issue.

**Comment by francisdb** [comment]
This should be fixed in v0.11
@rodricifuentes1 can you please confirm and close?

**Comment by ennru** [comment]
No update since July? I reckon this is outdated.
